CHANGELOG.md

```diff
@@ -4,13 +4,24 @@ Bob versions changelog
 
 ## [Unreleased]
 #### Added
+- Integration tests
+
+#### Changed
+
+
+#### Fixed
+
+
+#### Updated
+
+
+
+## [2.1.0-alpha.7] - 2022-11-28
+#### Added
 - Include bobt into the zip archive attached to the release infrastructure (#669)
 - Include bobc into release builds (#569)
 - Added blob-info and index-info features to brt (#356)
 - Support for files, file name patterns, key ranges and a 'exists' subcommand to 'bobc' (#539)
-- Integration tests (#518)
-- Added integration tests for aliens (#642)
-- Added integration tests for bobt (#648)
```

Comment:
Something bad happened here. Revert all this changes and add normal description of this particular PR